js/components/date.js

```diff
@@ -53,7 +51,7 @@ Fliplet.FormBuilder.field('date', {
 
     if (Fliplet.Env.get('platform') === 'web') {
       this.datePicker = $(this.$el).find('input.date-picker').datepicker({
-        format: 'yyyy-mm-dd',
+        format: 'yyyy/mm/dd',
```

Comment:
Why this format?

Reply:
> Why this format?

I used the function TD(date, {format: 'L'}) to translate the date. This function returns date in format 'yyyy/mm/dd'. So I had to change the format of DataPicker.

Reply:
Please do not include any date/number localisation in this work. This PR should only include work on translations.